packages/react-magma-dom/src/components/Table/TableRow.tsx

```diff
@@ -93,7 +95,7 @@ function buildTableRowColor(props) {
   return 'inherit';
 }
 
-const StyledTableRow = typedStyled.tr<{
+const StyledTableRow = styled.tr<{
```

Comment:
```suggestion
const StyledTableRow = typedStyled.tr<{
```

and all the instances below